lib/components/user/phone-number-editor.tsx

```diff
@@ -122,28 +138,57 @@ class PhoneNumberEditor extends Component<Props, State> {
     if (numberChanged && !initialPhoneNumberVerified) {
       this.setState({ phoneNumberReceived: true })
     }
+
+    // If a verification code was submitted successfully,
+    // i.e. initialPhoneNumber did not change AND initialPhoneVerified turns from false to true,
+    // set an ARIA alert that the phone number was successfully verified.
+    if (!numberChanged && verifiedChanged && initialPhoneNumberVerified) {
+      this.setState({ phoneNumberVerified: true })
+    }
   }
 
   render() {
-    const { initialPhoneNumber, onSubmitCode, phoneFormatOptions } = this.props
-    const { isEditing, phoneNumberReceived, submittedNumber } = this.state
+    const { initialPhoneNumber, phoneFormatOptions } = this.props
+    const {
+      isEditing,
+      phoneNumberReceived,
+      phoneNumberVerified,
+      submittedCode,
+      submittedNumber
+    } = this.state
     const hasSubmittedNumber = !isBlank(submittedNumber)
+    const hasSubmittedCode = !isBlank(submittedCode)
     const isPending = hasSubmittedNumber || this._isPhoneNumberPending()
-    const isPhoneChangeFormBusy = isEditing && hasSubmittedNumber
+    const isAlertBusy = (isEditing && hasSubmittedNumber) || hasSubmittedCode
+
+    // Note: ARIA alerts are read out only once, until they change,
+    // so there is no need to clear them.
+    // TODO: Find a correct way to render phone numbers for screen readers (at least for US).
+    let ariaAlertContent
+    if (phoneNumberReceived) {
+      ariaAlertContent = (
+        <FormattedMessage
+          id="components.PhoneNumberEditor.phoneNumberSubmitted"
+          values={{
+            phoneNumber: initialPhoneNumber
```

Comment:
Thanks for grouping together a lot of the functionality in the last two PRs! Would be awesome if we could address to-do in this PR too as [per discussion](https://jhalabi.com/blog/accessibility-phone-number-formatting). Regex/splicing methods could work here.